google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/SchemaAwareStreamWriter.java

```diff
@@ -141,21 +141,8 @@ private Message buildMessage(T item)
               .build();
       WriteStream writeStream = client.getWriteStream(writeStreamRequest);
       refreshWriter(writeStream.getTableSchema());
-      try {
```

Comment:
The previous comment "// Backend cache for GetWriteStream schema staleness can be 30 seconds, wait a bit before" need updates